packages/ui/src/proposals/components/ProposalFilters/ProposalFilters.tsx

```diff
@@ -40,6 +42,15 @@ const filterReducer = (filters: ProposalFiltersState, action: Action): ProposalF
         ...filters,
         [action.field]: typeof action.value == 'string' ? toCamelCase(action.value) : action.value,
       }
+
+    case 'update':
+      return {
+        search: toCamelCase(action.value.search) || '',
+        stage: toCamelCase(action.value.stage),
+        type: toCamelCase(action.value.type),
+        lifetime: action.value.lifetime,
```

Comment:
This fails because when saving filters to cookies lifetime gets converted from `Date` to `string`, so the page on load throws an error. 

In `update` action you will have to parse the string to `Date` object for Pioneer to work.

```suggestion
        lifetime:
          action.value.lifetime &&
          Object.entries(action.value.lifetime).reduce(
            (prev, [key, dateString]) => ({ ...prev, [key]: new Date(dateString) }),
            {} as DateRange
          ),
```

Reply:
Or at least it does for chrome.

![image](https://user-images.githubusercontent.com/92513933/196664635-b0cf97d6-f9b1-4855-a986-ddc09a7ef1ac.png)